pages/2017/index.rst

```diff
@@ -8,4 +8,60 @@ Python in Astronomy 2017
 Lorentz Center, Leiden
 ----------------------
 
-WOOOO YEAG
+The third edition of the Python in Astronomy Conference will be held at the
+Lorentz Center on the **8 - 12th May 2017**.
+
+As with the two previous Python in Astronomy conferences we aim to bring a wide
+variety of people who use, develop or teach people about Python packages in the
+context of all forms of Astronomy. The conference will include presentations,
+tutorials unconference sessions and sprints. As well as building the community
```

Comment:
add comma after tutorials (and also after sessions if you like oxford comma as much as me)